.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,70 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
```

Comment:
This is not a valid syntax, according to [upstream doc](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#on).

Reply:
It's a supported event to my knowledge.

<img width="1279" alt="Screenshot 2024-08-13 at 08 48 30" src="https://github.com/user-attachments/assets/3e4e0f4a-c2b6-4b76-89d1-dfa1ed66251d">
<img width="1279" alt="Screenshot 2024-08-13 at 08 49 05" src="https://github.com/user-attachments/assets/f2162a4c-baa8-41b4-839f-7f8dffeebeb4">

Reply:
This is a current doc https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onevent_nametypes

Reply:
\o I missed that. Thanks :+1: